autogen/agentchat/conversable_agent.py

```diff
@@ -884,10 +884,9 @@ async def a_generate_function_call_reply(
         return False, None
 
     def _str_for_tool_response(self, tool_response):
-        func_name = tool_response.get("name", "")
         func_id = tool_response.get("tool_call_id", "")
         response = tool_response.get("content", "")
-        return f"Tool call: {func_name}\nId: {func_id}\n{response}"
+        return f"Tool Call Id: {func_id}\n{response}"
```

Comment:
are we sure we want to remove the name of the function here?

Reply:
It's good to keep it here.

Reply:
+1

Reply:
Ok. Makes it a little convoluted to not set tool_response["name"] or delete it after calling this method. Or can let tool_responses be an internal format and oai_reply can delete tool_response["name"] for oai.

Preferences?

Reply:
I see the problem. Now I'm OK with this change.

Reply:
+1